resources/dicts/events/lifegen_events/murder.json

```diff
@@ -247,7 +247,7 @@
         "As you enter the elder's den, you greet v_c with a smile, and they smile back at you and ask for help with ticks. You oblige, moving forward despite v_c questioning if you need some sort of herbs for that. Silently, you snap their neck, a quick and painless kill as you dart out of the den before anyone sees."
     ],
     "murder medicinecat kitten": [
-        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, with deathberry juice dripping from their mouth.",
+        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, already having gone to StarClan.",
```

Comment:
this can be changed to match the previous one

"You encourage v_c to eat some shiny red berries you've found, and after eating them, v_c is later found having travelled to meet Silverpelt early."